Modules/ThirdParty/GDCM/src/gdcm/Source/DataStructureAndEncodingDefinition/gdcmTransferSyntax.cxx

```diff
@@ -211,7 +218,9 @@ bool TransferSyntax::IsLossless() const
     TSField == MPEG2MainProfile ||
     TSField == MPEG2MainProfileHighLevel ||
     TSField == MPEG4AVCH264HighProfileLevel4_1 ||
-    TSField == MPEG4AVCH264BDcompatibleHighProfileLevel4_1
+    TSField == MPEG4AVCH264BDcompatibleHighProfileLevel4_1 ||
+    TSField == HTJ2KLossless ||
+    TSField == HTJ2KRPCLLossless
```

Comment:
E.g. _IsLossless()_   will return `false` for _HTJ2KLossless_, isn't it? Something is wrong here.

P.S. Just FYI, I don't plan to step into this stuff (I don't use GDCM / GDCMIO for my app), sorry.